src/js/components/AppListStatusFilterComponent.jsx

```diff
@@ -33,18 +26,18 @@ var AppListStatusFilterComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      appsStatusesCount: getInitialAppStatusCount(),
+      appsStatusesCount: {},
```

Comment:
Since this is not a number, but an object representing each status' count, why dont we name this `appStatusCounters` or simply `appStatuses` ? I was kinda confused by this `Count` being an object... 
naming is hard and i'm a nitpicker – feel free to ignore this